components/leaflet-maps/ClustersMap.vue

```diff
@@ -142,26 +80,77 @@
      }
    },
    methods: {
+    fillMarkersArray(mapMarkers) {
+      this.markers = mapMarkers.map((markerData) => {
+        const lat = markerData.longLat[0];
+        const lng = markerData.longLat[1];
+
+        // Building the popup content as an HTML string
+        const popupContent = `
```

Comment:
Is this the best way to do this? I don't know Vue well enough to be certain but I would have thought we could render a custom vue component with all of this included for the popover and just pass the data in as props?

Reply:
ahh no. That is cool to leave as is. Thanks for the explanation Ruby. Appreciate it